packages/gatsby-theme-docs/utils/slug-pre-process.js

```diff
@@ -0,0 +1,5 @@
+function preProcessSlug(text) {
```

Comment:
I'm not particularly opinionated with naming. @nkuehn you tell me what you prefer here